src/common/location/Location.tsx

```diff
@@ -0,0 +1,20 @@
+import { Tooltip, useMediaQuery, Text } from '@chakra-ui/react';
+
+export default function Location({
+  short,
+  long,
+  alwaysShort,
+}: {
+  short?: String;
+  long: String;
+  alwaysShort: boolean;
```

Comment:
```suggestion
  alwaysShort?: boolean;
```
Then you can not pass the prop when you want it to be false like:
```jsx
<Location short={...} long={...} />
```
and then something like this when you want it to be true:
```jsx
<Location short={...} long={...} alwaysShort />
```